test/distributed/rpc/jit/test_dist_autograd_spawn.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+import unittest
+
+from torch.testing._internal.common_distributed import MultiProcessTestCase
+from torch.testing._internal.common_utils import TEST_WITH_ASAN, run_tests
+from torch.testing._internal.distributed.rpc.jit.dist_autograd_test import (
+    JitDistAutogradTest,
+)
+
+
+@unittest.skipIf(
+    TEST_WITH_ASAN, "Skip ASAN as torch + multiprocessing spawn have known issues"
+)
+class JitDistAutogradTestWithSpawn(MultiProcessTestCase, JitDistAutogradTest):
```

Comment:
nit: could it be named as "DistAutogradJitTestWithSpawn"?

Reply:
That sounds more semantic correct. 
But I am thinking about convenience for searching. If we type "DistAutograd", every time we have to choose between autocomplete options, "JitTest" and "Test".